mmv1/third_party/terraform/services/compute/resource_compute_region_instance_group_manager.go.tmpl

```diff
@@ -995,7 +1060,7 @@ func resourceComputeRegionInstanceGroupManagerUpdate(d *schema.ResourceData, met
 	}
 
 	// target size should use resize
```

Comment:
I forget any context on why we use resize- why would/wouldn't we use patch all of the time?

Reply:
Honestly speaking, I have no idea why did we update the targetSize with RESIZE api. There must be some reason, but I have no idea about that. Btw I modify my reply on the previous comments about updating targetSize using patch. It's all about a way we decided to go through at that time (4 months ago).

Reply:
If we can just call patch all the time that wouldn't be a bad idea to make the switch! I couldn't find clear docs about preferring one over the other on c.g.c.. We'd do that out of band in a separate PR, though.

Reply:
I am available to do any kind of help for  development if needed. Thanks.